packages/p2p-v2/src/utils/general.ts

```diff
@@ -0,0 +1,2 @@
+/** Function to transform a nickname to a 2-letter character. Most commonly used inside Avatars */
+export const getShortNickname = (nickname: string): string => nickname?.substring(0, 2).toUpperCase();
```

Comment:
i think this is the same as the one in utils/string

Reply:
ooof gonna remove this asap